addons/website_project/views/project_portal_project_task_template.xml

```diff
@@ -1,5 +1,41 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+    <template id="task_submitted" name=" Task Submitted">
+        <t t-call="website.layout">
+            <div class="oe_structure oe_empty h-100">
+                <div class="container d-flex flex-column justify-content-center h-100">
+                    <div class="row justify-content-center mb16">
+                        <t t-if="request.session.get('form_builder_model_model', '') == 'project.task'">
+                            <t t-set="task" t-value="request.website._website_form_last_record().sudo()"/>
+                        </t>
+                        <h1 class="text-center">
+                            <i class="fa fa-check-circle fa-1x text-success me-2" role="img" aria-label="Success" title="Success"/>
+                                <t t-if="task">
+                                    <span>
+                                    Your Task Number is
+                                    <a t-if="request.session.uid and task.project_id" t-attf-href="/my/task/#{task.id}">#<span t-field="task.id"/></a>
+                                    <t t-else="">#<span t-field="task.id"/></t>.
+                                    </span>
+                                </t>
+                        </h1>
+                        <h2 class="text-center">Thank you for contacting us, our team will get right on it!</h2>
+                        <div class="text-center">
+                            <a class="btn btn-primary" t-if="task and request.session.uid and task.project_id"  t-attf-href="/my/task/#{task.id}">View Task</a>
+                            <a class="btn btn-primary" href='/'>Go to the Homepage</a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </t>
+    </template>
+
+    <record id="aboutus_page" model="website.page">
```

Comment:
I'd change the id here to something more relevant

Reply:
I agree with @pkyriakou, we should give a better id since `About us` page is not really the content that we will display. 😅 

maybe `task_submitted_page`?